test/parallel_api/ranges/std_ranges_test.h

```diff
@@ -22,7 +22,9 @@
 #if _ENABLE_STD_RANGES_TESTING
 
 #include <oneapi/dpl/ranges>
+#if _ONEDPL_CPP20_SPAN_PRESENT
```

Comment:
Please, do not use internal macros in the tests. If necessary, define it in `test_config.h`.

Reply:
I think that the double definition also is not good... 
Actually, it is "a feature test macro".
M.b. does it make sense to use `__cpp_lib_span >= 202002L` directly in the tests?

Reply:
It's been fixed: c2ae4d75d092d820342fe4c5db7ddb3770f91e43